libs/components/modals/src/lib/modules/modal/modal-configuration.ts

```diff
@@ -7,10 +7,22 @@ import { Injectable } from '@angular/core';
 export class SkyModalConfiguration {
   public fullPage?: boolean;
   public size?: string;
+  /**
+   * @deprecated
+   */
   public ariaDescribedBy?: string;
+  /**
+   * @deprecated
+   */
   public ariaLabelledBy?: string;
   public ariaRole?: string;
+  /**
+   * @deprecated
+   */
   public tiledBody?: boolean;
+  /**
+   * @deprecated
```

Comment:
☝️ This is an internal file so no descriptions are needed for the deprecations. Marked as deprecated just so we know as we're developing in the source code.